examples/mnist-classification/README.md

```diff
@@ -0,0 +1,86 @@
+|              :linear |     50x4x4 - 800 |  1x1x1000 - 1000 | [1000] |          |        |        |            | [1000 800] |
+|                :relu |  1x1x1000 - 1000 |  1x1x1000 - 1000 |        |          |        |        |            |            |
+|             :dropout |  1x1x1000 - 1000 |  1x1x1000 - 1000 |        |          |        |        |            |            |
+|              :linear |  1x1x1000 - 1000 |      1x1x10 - 10 |   [10] |          |        |        |            |  [10 1000] |
+|             :softmax |      1x1x10 - 10 |      1x1x10 - 10 |        |          |        |        |            |            |
+Parameter count: 849780
+Classification accuracy: 0.9018
+Saving network to trained-network.nippy
+Classification accuracy: 0.9209
+Saving network to trained-network.nippy
+Classification accuracy: 0.9345
+Saving network to trained-network.nippy
+Classification accuracy: 0.9405
+Saving network to trained-network.nippy
+Classification accuracy: 0.9467
+Saving network to trained-network.nippy
+Classification accuracy: 0.9576
+Saving network to trained-network.nippy
+Classification accuracy: 0.9556
+Classification accuracy: 0.9587
+Saving network to trained-network.nippy
+Classification accuracy: 0.9559
+Classification accuracy: 0.9668
+Saving network to trained-network.nippy
+```
+
+You will see output about compilation and building the image sets. The program also starts a web server at [http://localhost:8091/](http://localhost:8091/) that hosts a page that updates live with results and information about the training process, including a confusion matrix visualization. After you see `Gate opened on http://localhost:8091`, go ahead and open a web page.
+
+After each training epoch the new network's classification accuracy is evaluated against a test dataset. If the new network is found to be the best so far (on that metric), then the trained model is saved to the file: `trained-network.nippy`.
+
+When you think it has run long enough, go ahead a stop the process.
+
+Get a new REPL up in the core namespace and run the `label-one` function:
+
+```
+mnist-classification.core> (label-one)
+Ensuring image data is built, and available on disk.
+Building dataset from folder: mnist/test
+{:answer 2, :guess 2}
+```
+
+A randomly selected handwritten digit from the test dataset is chosen, shown in a popup, and classified using the last exported network. Notably, the `label-one` function only uses functionality from base `cortex`, and none from the `experiment` framework. `cortex` as a library has many fewer dependencies, which simplifies deployment of systems that only need inference without the complexities of the experiment training machinery.
+
+## `train.sh`
+
+The included `train.sh` shell script uses a Makefile and java to run the entire process outside of Lein. It starts a lot faster, runs a bit faster, and hopefully hints at a more repeatable process.
```

Comment:
The `-nt` flag _could_ be used to see if a file is newer obviating the separate file.